solvebio/resource/dataset.py

```diff
@@ -99,9 +85,11 @@ def fields(self, name=None, **params):
                 'up fields')
 
         if name:
-            # construct the field's full_name if a field name is provided
-            return DatasetField.retrieve(
-                '/'.join([self['full_name'], name]))
+            fields = client.get(self.fields_url, params)
+            for i in fields['data']:
+                if i['name'] == name:
+                    result = DatasetField.retrieve(i['id'], **params)
+                    return result
```

Comment:
can we retrieve the field with only a single lookup? using the field name and the dataset id?